### PR TITLE
Write datasets api

### PR DESCRIPTION
**Pull request by AtreyeeS**
I do not know if its too much code in the example. Maybe we can remove it later.

### CODE REVIEW

File: src/text/2-package-subsections/datasets.tex [hunk 1]
```diff
@@ -1,32 +1,12 @@
...
-		SpectrumDataset      & counts    & background, edisp, exposure       & RegionGeom            & --                                       & cash          & 1D spectral extraction with the FoV background                                  \\ \hline
-		SpectrumDatasetOnOff & counts    & edisp, exposure                   & RegionGeom            & acceptance, acceptance\_off, counts\_off & wstat         & 1D spectral extractions using Reflected regions                                 \\ \hline
-		FluxPointsDataset    & flux      & None                              & None                  & --                                       & chi2          & Fitting of precomputed flux points                                              \\ \hline
-	\end{tabular}
-	\caption{Some caption.}
-	\label{table:datasets_types}
-\end{table*}
-
-Multiple datasets, or the same or different type, can be contained in a Datasets container. The Datasets class adds the likelihood from its constituent members to compute the full likelihood for a joint fitting.
```
Comment (adonath) [inline src/text/2-package-subsections/datasets.tex:30]: Why not keep this paragraph (and the one below as well)?

File: src/text/2-package-subsections/datasets.tex [hunk 2]
```diff
@@ -1,32 +1,12 @@
...
-
-The total number of predicted counts for a dataset are computed as the sum of the predicted counts from all source components, plus the expected counts from the residual hadronic background. In case of normal datasets, the predicted counts from the hadronic background are computed directly from the model in reconstructed energy and spatial coordinates, whereas for OnOff datasets, the background is estimated from real off counts. The predicted counts from a source are obtained by forward folding with the IRFs, assuming that the IRFs are independent of one another.
```
Comment (adonath) [inline src/text/2-package-subsections/datasets.tex:32]: Could keep this for now. We have to talk about the on-off case as well...

Comment (AtreyeeS) [reply src/text/2-package-subsections/datasets.tex:32]: Thanks @adonath ! I found the first version to verbose. Adapted the writing a bit to keep your comments.

**Review by adonath — APPROVED** [review]
Thanks @AtreyeeS, no further comments from my side...

**Pull request merged**